lighthouse-cli/bin.js

```diff
@@ -94,6 +94,12 @@ async function begin() {
     cliFlags.outputPath = 'stdout';
   }
 
+  // show deprecation message for disableDeviceEmulation
+  if (cliFlags.disableDeviceEmulation) {
+    log.warn('config', 'The "--disable-device-emulation" option is deprecated.' +
```

Comment:
we can make this pull double duty if we say `"--disable-device-emulation" has been removed in v5.`

Reply:
```js
log.warn('config', 'The "--disable-device-emulation" option is deprecated.' +
    ' Please use "--emulated-form-factor=none" instead.');
```